proto/path/qos/evm.proto

```diff
@@ -6,6 +6,16 @@ option go_package = "github.com/buildwithgrove/path/observation/qos";
 import "path/qos/jsonrpc.proto";
 import "path/metadata/metadata.proto";
 
+// EVMResponseInvalidReason defines why an endpoint response was rejected.
+// Current invalid response types (as of PR #164):
+//   1. EmptyResponse - endpoint returned no data
+//   2. UnmarshalErr - response failed to parse into expected format
+enum EVMResponseInvalidReason {
+	EVM_RESPONSE_INVALID_REASON_UNSPECIFIED = 0;
+	EVM_RESPONSE_INVALID_REASON_EMPTY = 1;      // No response data received
+	EVM_RESPONSE_INVALID_REASON_UNMARSHAL = 2;  // Response parsing failed
+}
+
 // EVMRequestObservations captures all observations made while serving a single EVM blockchain service request.
 message EVMRequestObservations {
```

Comment:
Note that the indent of `chain_id` here seems to be inconsistent with other parts

Reply:
Thanks, ran all `.proto` files through `buf`: it only generated a warning re package naming.
Added a TODO to address the proto files package names warnings.